Header header

uint16 obj_id
float64 obj_dx
uint8 obj_lane
float64 obj_vx
float64 obj_dy
float64 obj_ax
bool obj_in_lane
float64 obj_vy
float64 obj_timestamp